x-pack/plugins/alerting/server/application/rule/methods/create/create_rule.ts

```diff
@@ -150,8 +162,10 @@ export async function createRule<Params extends RuleParams = never>(
   const throttle = data.throttle ?? null;
 
   // Convert domain rule object to ES rule attributes
-  const ruleAttributes = transformRuleDomainToRuleAttributes(
-    {
+  const ruleAttributes = await transformRuleDomainToRuleAttributes({
```

Comment:
Just thinking. Transformation function sounds like synchronous function for me. Can it have downsides to keep it async?